info/vorlesungszeit/hopo.tex

```diff
@@ -0,0 +1,58 @@
+An der Uni mag es zwar manchmal so wirken, dass die Studierenden der unwichstigste
```

Comment:
Very good! Finally this in the Anfiheft.

Reply:
Only for you :kissing_heart: